html/index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+    <link
+      rel="stylesheet"
+      href="http://yui.yahooapis.com/3.18.1/build/cssreset/cssreset-min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body>
+    <div class="root">
+      <div class="checkout">
+        <div class="co-menu">
+          <div class="co__heading">Checkout</div>
+          <div id="breadcrum">
+            <ul>
+              <li>Shipping</li>
+              <li class="breadcrum--active">Payment Method</li>
+              <li>Confirmation</li>
+            </ul>
+          </div>
+          <div class="content-right">
+            <i class="fa fa-window-close"></i>
+          </div>
+        </div>
+        <div class="next-content">
+          <div class="co-paymentBlock">
+            <h4 class="next-content__heading">Wallet</h4>
```

Comment:
Please do not use underscore while defining classes in csss.
Also do not use upper case.